packages/extension-browser/src/devtools/views/controls/source-code.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+import SyntaxHighlighter from 'react-syntax-highlighter/dist/esm/prism-light';
+import css from 'react-syntax-highlighter/dist/esm/languages/prism/css';
+import javascript from 'react-syntax-highlighter/dist/esm/languages/prism/javascript';
+import xml from 'react-syntax-highlighter/dist/esm/languages/prism/markup';
+
+import * as styles from './source-code.css';
+
+// Explictly register languages so only those needed get bundled.
+SyntaxHighlighter.registerLanguage('css', css);
+SyntaxHighlighter.registerLanguage('javascript', javascript);
+SyntaxHighlighter.registerLanguage('xml', xml);
+
+type Props = {
+    children: any;
+    language?: string;
+};
+
+const SourceCode = ({ children, language }: Props) => {
+    return (
```

Comment:
Since we have to specify the default language as html twice, I think it's worth moving to a variable.

```suggestion
    const lang = language || 'html';

    return (
```